wagtail/admin/templates/wagtailadmin/pages/edit.html

```diff
@@ -2,13 +2,30 @@
 {% load wagtailadmin_tags %}
 {% load i18n %}
 {% load l10n %}
+{% load tabs %}
 {% block bodyclass %}editor-view {% if page.live %}page-is-live{% endif %} {% if page_locked %}content-locked{% endif %}{% endblock %}
 
 {% block content %}
     {% page_permissions page as page_perms %}
 
     {% include 'wagtailadmin/shared/headers/page_edit_header.html' with title=header_title %}
 
+    {% tabs %}
```

Comment:
Why are we adding this here? Is this just for an example?

There is a style guide page specifically for code examples and design checks. Maybe review the tabs on that page and migrate them to this new approach.